config/crd/kustomization.yaml

```diff
@@ -4,19 +4,25 @@
 resources:
 - bases/core.openfeature.dev_featureflagconfigurations.yaml
 - bases/core.openfeature.dev_flagsourceconfigurations.yaml
+- bases/core.openfeature.dev_featureflags.yaml
+- bases/core.openfeature.dev_featureflagsources.yaml
 #+kubebuilder:scaffold:crdkustomizeresource
 
 patchesStrategicMerge:
 # [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix.
 # patches here are for enabling the conversion webhook for each CRD
 - patches/webhook_in_featureflagconfigurations.yaml
 - patches/webhook_in_flagsourceconfigurations.yaml
+#- patches/webhook_in_featureflags.yaml
+#- patches/webhook_in_featureflagsources.yaml
 #+kubebuilder:scaffold:crdkustomizewebhookpatch
 
 # [CERTMANAGER] To enable cert-manager, uncomment all the sections with [CERTMANAGER] prefix.
 # patches here are for enabling the CA injection for each CRD
 - patches/cainjection_in_featureflagconfigurations.yaml
 - patches/cainjection_in_flagsourceconfigurations.yaml
+#- patches/cainjection_in_featureflags.yaml
```

Comment:
why is the CA injection disabled for this version?

Reply:
was the default option when creating CRDs and not creating dedicated controllers for them. If we need it in the future, we can enable it. For now I would say it's ok